recipes/vs2008_runtime/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: vs2008_runtime
+  version: "9.0.30729.6161"
+
+build:
+  number: 0
+  skip: true          # [not win]
+  msvc_compiler: 9.0
+
+test:
+  commands:
+    {% set libs = [
+        "msvcm90",
+        "msvcp90",
+        "msvcr90",
+        "vcomp90"
+    ] %}
+    {% for each_lib in libs %}
+    - if not exist "%PREFIX%\\{{ each_lib }}.dll" exit 1
+    - if not exist "%PREFIX%\\Library\\bin\\{{ each_lib }}.dll" exit 1
+    {% endfor %}
+
+about:
+  home: http://www.microsoft.com
+  license: Proprietary
+  summary: Bundles of the MSVC 9 (VS 2008) runtime
+
+extra:
+  recipe-maintainers:
+    - gillins
+    - isuruf
```

Comment:
Went ahead and added you as a maintainer, @isuruf. Please let me know if that is ok.

Reply:
Yes